forge-gui/res/quest/precons/Devour for Power.dck

```diff
@@ -0,0 +1,90 @@
+[metadata]
+Name=Devour for Power
+[shop]
+WinsToUnlock=0
+Credits=5000
+MinDifficulty=0
+MaxDifficulty=5
+[metadata]
+Description=When The Mimeoplasm becomes your commander, death becomes your friend. Opponents will watch in helpless terror as you devour creatures from one graveyard to help you pound creatures into another. It’s a fiendish carnival of cadavers, and you’ll be the ringmaster!
```

Comment:
`It’s a fiendish carnival of cadavers, and you’ll` -> `It's a fiendish carnival of cadavers, and you'll`